group_vars/customer/eugene/ci/main.yml

```diff
@@ -3,3 +3,8 @@ redirections:
   - to: "hello.{{ project_name}}.{{ domain_name }}"
     from:
       - "www.hello.{{ project_name}}.{{ domain_name }}"
+
+# Richie
+# Test extra dependencies installation
+richie_extra_dependencies:
+  - "factory-boy[django]>=2.11.1"
```

Comment:
maybe add also `Faker` if we want `eugene` to be operational?

Reply:
No it's just for testing purpose.